src/proto/mgmt/svc.proto

```diff
@@ -121,4 +121,17 @@ message PoolMonitorReq {
 	string poolUUID = 1;	// Pool UUID associated with the Pool Handle
 	string poolHandleUUID = 2; // Pool Handle UUID for the connection
 	string jobid = 3;	// Job ID to associate instance with.
+}
+
+message HandleList {
```

Comment:
Might be nice to call this something like PoolHandles. Or alternately make the HandleList generic, with the members being something like resourceUUID and handles.

Reply:
Unless you plan to use this message type in other requests, it would probably be better to move it into PoolHandleCleanupReq. If you do plan to use it in other requests, then I agree with Kris that it would be better to name it PoolHandleList to make its purpose clear.

Reply:
Removed these messages all together.